common/inc/stdc++/linux/exception

```diff
@@ -33,11 +33,11 @@ namespace std
 	class exception
 	{
 	public:
-		exception() throw();
```

Comment:
stdexcept.cc and exception.cc in sdk/cpprt/linux should be changed according to the header changes. Thanks.